patches/minecraft/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.java.patch

```diff
@@ -1,5 +1,13 @@
 --- a/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.java
 +++ b/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.java
+@@ -49,7 +_,6 @@
+ 
+ @OnlyIn(Dist.CLIENT)
+ public class CreativeModeInventoryScreen extends EffectRenderingInventoryScreen<CreativeModeInventoryScreen.ItemPickerMenu> {
+-   private static final ResourceLocation f_98504_ = new ResourceLocation("textures/gui/container/creative_inventory/tabs.png");
```

Comment:
Patching this out seems unnecessary, even if it is now unused.

Reply:
Reverted

Reply:
Awesome